manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.installer.yaml

```diff
@@ -9,4 +9,4 @@ Installers:
   InstallerUrl: https://github.com/LukeBriggsDev/Pepys/releases/download/v1.5.3/pepys.1.5.3.win.x86_64.installer.exe
   InstallerSha256: 902ce9565d6c815e580ea35045c1223c79f6a876ff2a9f89e5b9662d1593885a
 ManifestType: installer
-ManifestVersion: 1.0.0
\ No newline at end of file
+ManifestVersion: 1.0.1
```

Comment:
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

Reply:
Also, 1.0.1 schema doesn't exist.

We're at 1.1.0 now.